eigenpro/optimizers.py

```diff
@@ -45,9 +45,8 @@ def __init__(self,
         else:
             self.grad_accumulation = None
 
-        #### adding nystrom samples to the model
-        self._model.add_centers(data_preconditioner.centers.to(dtype), None,
-                                nystrom_centers = True)
+        #### Initilizing nystrom samples to the model
```

Comment:
use `#` instead of `####` for comments to achieve consistency and follow the python style guide.